vud.theme.inc

```diff
@@ -400,15 +423,15 @@ function vud_vote($type, $entity_id, $value, $tag, $widget, $token, $ajax = 'aja
   if (!is_numeric($value) || !drupal_valid_token($token, "vote/$type/$entity_id/$value/$tag/$widget", TRUE)) {
     return MENU_ACCESS_DENIED;
   }
-
+  
   $vote = array();
   $casted_vote_criteria = array(
-    'entity_type' => $type,
-    'entity_id' => $entity_id,
-    'tag' => $tag,
-  ) + votingapi_current_user_identifier();
+      'entity_type' => $type,
+      'entity_id' => $entity_id,
+      'tag' => $tag,
+    ) + votingapi_current_user_identifier();
   $casted_vote = votingapi_select_single_vote_value($casted_vote_criteria);
-
+  
```

Comment:
Spaces without a real line are against coding standards, please remove this unneeded and wrong change.
More below, please fix them